### ISSUE TITLE
Insert error "read from fail"

### ISSUE THREAD

**Issue by chelsEg** [lizhichao/one-ck#4]
Hello.

I have some interesting error after call insert function.

**Tables structure:**
```
CREATE TABLE IF NOT EXISTS categories_replica ON CLUSTER '{cluster}' (
    id UInt64,
    value String,
    category String,
    group Nullable(String) DEFAULT NULL
) engine = ReplicatedMergeTree(
  '/clickhouse/{installation}/{cluster}/tables/{shard}/{database}/categories_replica',
  '{replica}'
) ORDER BY id

CREATE TABLE IF NOT EXISTS categories ON CLUSTER '{cluster}' AS categories_replica
ENGINE = Distributed('{cluster}', '{database}', categories_replica, rand())
```
**Inserting data:**
```
Array
(
    [0] => Array
        (
            [id] => 1
            [value] => 100
            [category] => banner_cluster
            [group] => NULL
        )
    [1] => Array
        (
            [id] => 2
            [value] => viber
            [category] => messenger
            [group] => viber
        )
    [2] => Array
        (
            [id] => 5
            [value] => normal
            [category] => body_type
            [group] => normal
        )
)
```
**Php code:**
`$client->insert('categories', $data);`

After call function _insert_ data was **inserted** in ClickHouse, but function exit with error!

**Error trace:**
```
local.ERROR: read from fail {"exception":"[object] (OneCk\\CkException(code: 10001): read from fail at /application/vendor/lizhichao/one-ck/src/Read.php:21)
[stacktrace]
#0 /application/vendor/lizhichao/one-ck/src/Read.php(24): OneCk\\Read->getChar(16)
#1 /application/vendor/lizhichao/one-ck/src/Read.php(76): OneCk\\Read->getChar(73)
#2 /application/vendor/lizhichao/one-ck/src/Client.php(364): OneCk\\Read->string()
#3 /application/vendor/lizhichao/one-ck/src/Client.php(108): OneCk\\Client->readErr()
#4 /application/vendor/lizhichao/one-ck/src/Client.php(344): OneCk\\Client->receive()
#5 /application/vendor/lizhichao/one-ck/src/Client.php(272): OneCk\\Client->writeEnd()
#6 /application/vendor/illuminate/support/Facades/Facade.php(261): OneCk\\Client->insert('categories', Array)
```

**Comment by lizhichao** [comment]
What is your clickhouse version? I did not reproduce it. Or you give a minimal reproducible example

**Comment by chelsEg** [comment]
> What is your clickhouse version? I did not reproduce it. Or you give a minimal reproducible example

**ClickHouse version:** 20.8.6.6

**My full code for reproduce**:
```
$result = $client->query("SELECT id, value, category, group FROM categories WHERE id IN (1, 2, 5)");

if (empty($result)) {
    $client->insert('categories', $data);
}
```

**Comment by chelsEg** [comment]
> What is your clickhouse version? I did not reproduce it. Or you give a minimal reproducible example

I'm found dependency!

**Reproducing workflow:**

- Run _query_ method;
- Run _insert_ method;
- Bug is reproduced.

If you run only _insert_ method, all work perfectly.

**Comment by lizhichao** [comment]
Thank you for finding the problem， It's fixed 0.2.2 tag

**Comment by chelsEg** [comment]
> Thank you for finding the problem， It's fixed 0.2.2 tag
No problem 😉 

I'm use your library very strong!

All works perfect!
